
{'date': 'Sun Jul 22 2018 20:50:22.982 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 20:55:32.776 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 158
    Tabuleiro ()
  module <module> line 27
    self.linhaA1 = Elemento(QBRANCO, style = dict(
NameError: name 'QBRANCO' is not defined
'''},
{'date': 'Sun Jul 22 2018 21:02:33.383 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 162
    Tabuleiro ()
  module <module> line 26
    self.fase1 = Cena(img = FASE1, style = dict(
  module _spy.vitollino.main line 696
    Cena.c(**kwargs)
  module _spy.vitollino.main line 739
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},
{'date': 'Sun Jul 22 2018 21:02:34.588 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 162
    Tabuleiro ()
  module <module> line 26
    self.fase1 = Cena(img = FASE1, style = dict(
  module _spy.vitollino.main line 696
    Cena.c(**kwargs)
  module _spy.vitollino.main line 739
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},
{'date': 'Sun Jul 22 2018 21:03:52.56 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 158
    Tabuleiro ()
  module <module> line 26
    self.fase1 = Cena(img = FASE1)
  module _spy.vitollino.main line 698
    self._cria_divs(width)
  module _spy.vitollino.main line 703
    divesq.style.width = width // 3  # 100
TypeError: unsupported operand type(s) for //: 'str' and 'int'
'''},
{'date': 'Sun Jul 22 2018 21:36:20.999 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 27
  self.linhaA1 = Elemento(QBRANCO, style = dict(
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jul 23 2018 19:31:27.249 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 190
  Desafio_Cor_Fase1 = ["0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0"]
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jul 23 2018 19:32:23.758 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 190
  Desafio_Cor_Fase1 = ["0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0"]
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jul 23 2018 19:32:30.594 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 190
  Desafio_Cor_Fase1 = ["0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0"]
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jul 23 2018 19:33:04.88 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 254
    Tabuleiro ()
  module <module> line 203
    self.escolha_cria()
AttributeError: 'Tabuleiro' object has no attribute 'escolha_cria'
'''},
{'date': 'Mon Jul 23 2018 19:34:01.382 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 193
  self.escolha_cria(self):
                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jul 23 2018 19:34:31.793 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 193
  self.escolha_cria():
                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jul 23 2018 19:34:45.449 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 193
  self.escolha():
                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jul 23 2018 19:35:02.829 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 194
  alert(self.tabuleiro[nome].elt.onclick)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jul 23 2018 19:39:37.957 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 254
    Tabuleiro ()
  module <module> line 202
    self.tabuleiro[nome].elt.onclick = alert(self.tabuleiro[nome])         
AttributeError: 'Tabuleiro' object has no attribute 'tabuleiro'
'''},
{'date': 'Mon Jul 23 2018 19:42:36.473 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 255
    Tabuleiro ()
  module <module> line 203
    self.tabuleiro[nome].elt.onclick = recoloca_clique_aqui         
AttributeError: 'Tabuleiro' object has no attribute 'tabuleiro'
'''},
{'date': 'Mon Jul 23 2018 21:00:41.694 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 152
    Tabuleiro ()
  module <module> line 101
    self.escolha_cria()
AttributeError: 'Tabuleiro' object has no attribute 'escolha_cria'
'''},
{'date': 'Mon Jul 23 2018 21:01:13.786 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 152
    Tabuleiro ()
  module <module> line 101
    self.escolha_cria()
AttributeError: 'Tabuleiro' object has no attribute 'escolha_cria'
'''},
{'date': 'Mon Jul 23 2018 21:05:10.94 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 154
    Tabuleiro ()
  module <module> line 31
    alert(Desafio_Cor_Fase1[1,1])
TypeError: list indices must be integer, not tuple
'''},
{'date': 'Mon Jul 23 2018 21:31:19.856 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 181
    Tabuleiro ()
  module <module> line 85
    self.corAB2() 
AttributeError: 'Tabuleiro' object has no attribute 'corAB2'
'''},
{'date': 'Mon Jul 23 2018 21:47:42.140 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 162
    Tabuleiro ()
  module <module> line 33
    self.linhaA1 = Elemento(clicado, style = dict(left= 160, top= "127px", width=101, height="74px"))
NameError: name 'clicado' is not defined
'''},
{'date': 'Mon Jul 23 2018 21:48:21.769 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    Tabuleiro ()
  module <module> line 69
    colorir()
TypeError: colorir() missing 1 positional argument: self
'''},
{'date': 'Mon Jul 23 2018 21:50:34.704 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 164
    Tabuleiro ()
  module <module> line 42
    self.linhaA1 = Elemento(clicado, style = dict(left= 160, top= "127px", width=101, height="74px"))
NameError: name 'clicado' is not defined
'''},
{'date': 'Mon Jul 23 2018 22:02:55.330 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 169
    Tabuleiro ()
  module <module> line 29
    self.fase1 = Cena(img = FASE1)
  module _spy.vitollino.main line 698
    self._cria_divs(width)
  module _spy.vitollino.main line 703
    divesq.style.width = width // 3  # 100
TypeError: unsupported operand type(s) for //: 'str' and 'int'
'''},
{'date': 'Mon Jul 23 2018 23:15:08.690 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 167
    Tabuleiro ()
  module <module> line 32
    quadros = [linhaA1,linhaB1,linhaC1,linhaA2,linhaB2,linhaC2,linhaA3,linhaB3,linhaC3,linhaA4,linhaB4,linhaC4]
NameError: name 'linhaA1' is not defined
'''},
{'date': 'Mon Jul 23 2018 23:21:27.806 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 40
  for quadro in range(3)):
                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jul 23 2018 23:21:42.825 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 168
    Tabuleiro ()
  module <module> line 32
    quadros = [linhaA1,linhaB1,linhaC1,linhaA2,linhaB2,linhaC2,linhaA3,linhaB3,linhaC3,linhaA4,linhaB4,linhaC4]
NameError: name 'linhaA1' is not defined
'''},
{'date': 'Mon Jul 23 2018 23:23:51.591 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    Tabuleiro ()
  module <module> line 32
    quadros = [linhaA1,linhaB1,linhaC1,linhaA2,linhaB2,linhaC2,linhaA3,linhaB3,linhaC3,linhaA4,linhaB4,linhaC4]
NameError: name 'linhaA1' is not defined
'''},
{'date': 'Mon Jul 23 2018 23:26:46.873 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 166
    Tabuleiro ()
  module <module> line 115
    colorir(self)
  module <module> line 41
    self.quadros[quadro] = Elemento(Desafio_Cor_Fase1[quadro], style = dict(left= 114+lef, top= "148px", width=80, height="80px"))
AttributeError: 'Tabuleiro' object has no attribute 'quadros'
'''},
{'date': 'Mon Jul 23 2018 23:31:53.140 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 157
    Tabuleiro ()
  module <module> line 106
    colorir(self)
NameError: name 'colorir' is not defined
'''},
{'date': 'Mon Jul 23 2018 23:33:31.55 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 157
    Tabuleiro ()
  module <module> line 57
    self.linhaC4 = Elemento(Cor_Fase1[12], style = dict(left= 288, top= "390px", width=80, height="80px"))
IndexError: list index out of range
'''},
{'date': 'Mon Jul 23 2018 23:52:16.891 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 109
  self.fase1.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jul 23 2018 23:52:36.581 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
Exception: self.$events is undefined
  module '$exec_8873' line 158
Tabuleiro ()
'''},
{'date': 'Tue Jul 24 2018 01:27:29.958 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
Exception: self.$events is undefined
  module '$exec_8931' line 161
Tabuleiro ()
'''},
{'date': 'Tue Jul 24 2018 01:28:22.487 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
Exception: self.$events is undefined
  module '$exec_8989' line 161
Tabuleiro()
'''},
{'date': 'Tue Jul 24 2018 01:30:36.189 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 161
    Tabuleiro (self)
TypeError: __init__() takes 1 positional argument but more were given
'''},
{'date': 'Tue Jul 24 2018 01:31:08.222 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
Exception: self.$events is undefined
  module '$exec_9052' line 161
Tabuleiro ()
'''},
{'date': 'Tue Jul 24 2018 01:31:36.652 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 161
    Tabuleiro ()
  module <module> line 95
    self.linhaA1.elt.onclick = escolha_cria(self.fase1)
  module <module> line 30
    self.linhaA1.entra(self.fase1)
AttributeError: 'Cena' object has no attribute 'linhaA1'
'''},
{'date': 'Tue Jul 24 2018 01:57:10.904 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 48
    self.tabela_fase1 = tabelafase1 = Cena(img=TABELAFASE1)
NameError: name 'TABELAFASE1' is not defined
'''},
{'date': 'Tue Jul 24 2018 01:57:42.460 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 49
    self.pilha = Elemento(ALIMENTO_FASE1_1, tit='Alimento1', style=dict(
NameError: name 'ALIMENTO_FASE1_1' is not defined
'''},
{'date': 'Tue Jul 24 2018 01:58:12.726 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 49
    self.pilha = Elemento(ALIMENTO_FASE1_1, tit='Alimento1', style=dict(
NameError: name 'ALIMENTO_FASE1_1' is not defined
'''},
{'date': 'Tue Jul 24 2018 01:58:33.782 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 53
    Pilha_Cartas = [EXCREMENTO_FASE1_16, DIVERSAO_FASE1_15, DESCANSO_FASE1_14, ALIMENTO_FASE1_13,\
NameError: name 'EXCREMENTO_FASE1_16' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:03:42.387 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 66
    self.casa0 = Elemento(RESP_SORRISOMAIS_FASE1, tit='0_0', style=dict(
NameError: name 'RESP_SORRISOMAIS_FASE1' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:04:20.469 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 68
    self.casa = Elemento(RESP_SORRISOMAIS_FASE1, tit='0_1', style=dict(
NameError: name 'RESP_SORRISOMAIS_FASE1' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:04:39.726 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 76
    self.tabuleiro_respostas[nome] = Elemento(FUNDO_BRANCO, tit=nome+"_", style=dict(
NameError: name 'FUNDO_BRANCO' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:05:40.40 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 89
    self.cliqueaqui = Elemento (CLIQUEAQUI, style=dict(width="170px", height="100px", left=30, top=30))
NameError: name 'CLIQUEAQUI' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:06:03.537 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 94
    self.casa0 = Elemento(ALIMENTO_FASE1_1, tit='0_0', style=dict(
NameError: name 'ALIMENTO_FASE1_1' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:06:20.422 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 226
    Tabuleiro()
  module <module> line 103
    self.tabuleiro[nome] = Elemento(FUNDO_VAZIO, tit=nome+"_", style=dict(
NameError: name 'FUNDO_VAZIO' is not defined
'''},
{'date': 'Tue Jul 24 2018 02:30:29.222 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 195
    Tabuleiro()
  module <module> line 99
    self.cliqueaqui.elt.onclick = remove_clique_aqui        
AttributeError: 'Tabuleiro' object has no attribute 'cliqueaqui'
'''},
{'date': 'Tue Jul 24 2018 02:35:56.512 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 203
    Tabuleiro()
  module <module> line 107
    self.cliqueaqui.elt.onclick = remove_clique_aqui        
AttributeError: 'Tabuleiro' object has no attribute 'cliqueaqui'
'''},
{'date': 'Tue Jul 24 2018 02:44:13.704 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 203
    Tabuleiro()
  module <module> line 73
    self.cliqueaqui.entra (tabelafase1)
AttributeError: 'Tabuleiro' object has no attribute 'cliqueaqui'
'''},
{'date': 'Tue Jul 24 2018 02:58:37.448 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 204
    Tabuleiro()
  module <module> line 87
    self.tabuleiro[nome] = Elemento(FUNDO_BRANCO, tit=nome+"_", style=dict(
UnboundLocalError: local variable 'coluna' referenced before assignment
'''},
{'date': 'Tue Jul 24 2018 11:57:59.920 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
Exception: self.$events is undefined
  module '$exec_7836' line 115
Tabuleiro()
'''},
{'date': 'Tue Jul 24 2018 11:58:13.275 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
Exception: self.$events is undefined
  module '$exec_7881' line 115
Tabuleiro()
'''},
{'date': 'Sun Sep 23 2018 23:48:47.832 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 18
    simetria1 = box(pos=(4.2, -2, 0), size=(1, 1, 1), **bloco)
NameError: name 'bloco' is not defined
'''},